lib/api/src/js/instance.rs

```diff
@@ -106,24 +105,43 @@ impl Instance {
     pub fn from_module_and_instance(
         mut store: &mut impl AsStoreMut,
         module: &Module,
-        instance: WebAssembly::Instance,
+        externs: Vec<Extern>,
```

Comment:
That looks like a breaking API change.

Reply:
We probably don't want this

Reply:
this was to reduce code duplication here but we could try and take it out - wasmer-js is working

Reply:
@john-sharratt I already started removing this one!